src/packages/solid/components/ProgressBar/ProgressBar.tsx

```diff
@@ -15,52 +15,28 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-import type { Component } from 'solid-js';
+import { type Component } from 'solid-js';
 import { View } from '@lightningjs/solid';
-import type { UIComponentProps } from '../../types/interfaces.js';
 import styles from './ProgressBar.styles.js';
-
-export interface ProgressBarProps extends UIComponentProps {
-  /**
-   * a numeric value of the current progress represented as a decimal between 0 and 1
-   */
-  progress: number;
-
-  /**
-   * color of the overlay portion of the progress bar
-   */
-  progressColor?: string | number;
-  /**
-   * total width of the component
-   */
-  width: number;
-  /**
-   * total height of the component
-   */
-  height?: number;
-}
+import type { ProgressBarProps } from './ProgressBar.types.js';
 
 const ProgressBar: Component<ProgressBarProps> = (props: ProgressBarProps) => {
   return (
     <View
       {...props}
-      forwardStates
+      borderRadius={props.radius}
+      color={props.barColor}
       // @ts-expect-error TODO type needs to be fixed in framework
-      style={[
-        props.style, //
-        styles.Container.tones[props.tone || styles.tone],
-        styles.Container.base
-      ]}
+      style={[props.style, styles.Container.tones[props.tone ?? styles.tone], styles.Container.base]}
+      forwardStates
     >
       <View
-        forwardStates
+        borderRadius={props.radius}
         animationSettings={props.animationSettings}
         width={props.width * props.progress}
         color={props.progressColor}
-        style={[
-          styles.ProgressBar.tones[props.tone || styles.tone], //
-          styles.ProgressBar.base
-        ]}
+        style={[styles.ProgressBar.tones[props.tone ?? styles.tone], styles.ProgressBar.base]}
```

Comment:
can we restore the // to the two style properties in this component? i just think it makes the arrays easier to read